src/main/scripts/container/Dockerfile

```diff
@@ -7,6 +7,7 @@ ADD ./entrypoint /bin/entrypoint
 
 ENV SCRIPTS_HOME /scripts
 ENV JBANG_VERSION @projectVersion@
+ENV JBANG_NO_VERSION_CHECK true
```

Comment:
why setting it here ? that will make jbang github action invisible.

Reply:
I don't understand what you're trying to say?

Reply:
this dockerfile is what jbang github action uses. not for testin